website/content/docs/nia/configuration.mdx

```diff
@@ -373,19 +381,19 @@ task {
     // ...
   }
 
-  source_input "<input-type>" {
+  module_input "<input-type>" {
     // ...
   }
 }
 ```
 
-~> The type of the `source_input` block that can be configured depends on the `condition` block type and the `services` field (deprecated). See [Task Source Input Restrictions](/docs/nia/configuration#task-source-input-restrictions) for more details.
+~> The type of the `module_input` block that can be configured depends on the `condition` block type and the `services` field (deprecated). See [Task Module Input Restrictions](/docs/nia/configuration#task-module-input-restrictions) for more details.
 
-The following sections describe the source input types that CTS supports.
+The following sections describe the module input types that CTS supports.
 
-#### Services Source Input
+#### Services Module Input ((#services-source-input))
```

Comment:
What does the #services-source-input do?

Reply:
It's an anchor link alias so that this header will have two anchor links `#services-module-input` and `#services-source-input`. That way any bookmarks for the old header containing "source input" won't break